packages/components/stories/iconography.stories.mdx

```diff
@@ -0,0 +1,51 @@
+<!--This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+OpenCRVS is also distributed under the terms of the Civil Registration
+& Healthcare Disclaimer located at http://opencrvs.org/license.
+
+Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+graphic logo are (registered/a) trademark(s) of Plan International.-->
+
+import { Meta, IconGallery, IconItem } from '@storybook/addon-docs'
+import { Icon } from '../src/Icon'
+import * as formIcons from '../src/Icon/form-icons'
+import * as icons from '../src/Icon/icons'
+import * as featherIcons from 'react-feather'
+
+<Meta title="Styles/Iconography" />
+
+export function AllIcons({ icons: allIcons }) {
+  return allIcons.map((iconName) => (
+    <IconItem name={iconName} key={iconName}>
+      <Icon name={iconName} />
+    </IconItem>
+  ))
+}
+
+# Iconography
+
+OpenCRVS UI-Kit is highly relying on open source [Feather icons](https://github.com/feathericons/feather). In addition to feather icons we have a few custom ones; form icons and custom ui-kit icons. The color of these icons cannot be changed with the `color` -prop.<br />
+For more information how to use the [&lt;Icon&gt;](/docs/typography-icon--default) component, see its story page.
+
+## Form icons
+
+<br />
+<IconGallery>
+  <AllIcons icons={Object.keys(formIcons)} />
+</IconGallery>
+
+## Custom UI-kit icons
+
+<br />
+<IconGallery>
+  <AllIcons icons={Object.keys(icons)} />
+</IconGallery>
```

Comment:
@jpye-finch do you think we should separate form icons / custom icons here either then? no?
![image](https://user-images.githubusercontent.com/1322608/197157360-a4e85dad-e4a2-4f3a-804a-645653931745.png)

Reply:
@naftis The sent icon should also be green right?

Reply:
Might be nice to split out form icons.

But we can put the rest under the heading Icons. No need to mention  feather icons imo